From 7bcabd15b2be28b00c564455643c3bb05d67091b Mon Sep 17 00:00:00 2001
From: Wei Yongjun <weiyj.lk@gmail.com>
Date: Sat, 20 Aug 2016 15:31:05 +0000
Subject: [PATCH 1736/2241] clk: mvebu: Remove redundant dev_err call in
 armada_3700_periph_clock_probe()

There is a error message within devm_ioremap_resource
already, so remove the dev_err call to avoid redundant
error message.

Change-Id: Iae3c03e51920a0bd54c0c6fb058f35cec2eeebe7
Signed-off-by: Wei Yongjun <weiyj.lk@gmail.com>
Signed-off-by: Stephen Boyd <sboyd@codeaurora.org>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40809
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 drivers/clk/mvebu/armada-37xx-periph.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/clk/mvebu/armada-37xx-periph.c b/drivers/clk/mvebu/armada-37xx-periph.c
index 5bb13c9..45905fc 100644
--- a/drivers/clk/mvebu/armada-37xx-periph.c
+++ b/drivers/clk/mvebu/armada-37xx-periph.c
@@ -383,10 +383,8 @@ static int armada_3700_periph_clock_probe(struct platform_device *pdev)
 
 	res = platform_get_resource(pdev, IORESOURCE_MEM, 0);
 	reg = devm_ioremap_resource(dev, res);
-	if (IS_ERR(reg)) {
-		dev_err(dev, "Could not map the periph clock registers\n");
+	if (IS_ERR(reg))
 		return PTR_ERR(reg);
-	}
 
 	driver_data = devm_kzalloc(dev, sizeof(*driver_data), GFP_KERNEL);
 	if (!driver_data)
-- 
2.7.4

